provider.go

```diff
@@ -32,6 +32,18 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("ELASTICSEARCH_URL", nil),
 				Description: "Elasticsearch URL",
 			},
+			"sniff": &schema.Schema{
```

Comment:
Can you pull this out into a separate PR? Happy to get this merged quickly.

Reply:
The Sniff and Healthcheck support? I will. It's useful on a setup where the nodes aren't routable :+1: